trio/_highlevel_socket.py

```diff
@@ -125,15 +130,15 @@ async def send_eof(self):
             with _translate_socket_errors_to_stream_errors():
                 self.socket.shutdown(tsocket.SHUT_WR)
 
-    async def receive_some(self, max_bytes=None):
+    async def receive_some(self, max_bytes: int | None = None):
```

Comment:
return type?

Reply:
oops, missed that. Is it just `Any`?

Reply:
typed as `-> bytes`, see https://github.com/python-trio/trio/pull/2682#discussion_r1252424862